projects/memapCore/src/main/java/memap/helper/milp/MILPProblemNoConnections.java

```diff
@@ -295,7 +295,41 @@ public LpSolve createSOCBoundaries(LpSolve problem, BuildingMessage buildingMess
 			if (maxChargeCapacity >= sm.capacity) {
 				maxChargeCapacity = sm.capacity;
 			}
+			
+			// Hard coded standby losses
+			double standbyLosses = 0.2;
```

Comment:
shall we take this value from the storage messages and remove the hard coded stuff?

Reply:
As soon as the input field in the GUI is implemented, yes. Would be great if you could do this once the GUI is adjusted.

Reply:
Denis is right. Both the GUI CSVStorage and the OPC UA ClientStorage send the storage message to the optimizer, so it should be read from the storage message. 
As long it is not implemented in the frontend, it should be rather hardcoded in the CSVStorage class and then fed into the storage message there.